src/App.js

```diff
@@ -2,17 +2,21 @@ import { BrowserRouter, Route, Routes } from "react-router-dom"
 import "./App.css"
 import Nav from "./Components/Nav"
 import Bookshelf from "./Components/Bookshelf"
-// import NotFound from './Components/NotFound';
+import NotFound from "./Components/NotFound"
 
 function App() {
     return (
         <div className="App">
             <BrowserRouter>
                 <Nav />
                 <Routes>
-                    <Route path="/" element={<Bookshelf />} />
-                    {/* <Route path="/book/:id" element={<Book />} />
-                    <Route path='*' element={<NotFound />} /> */}
+                    <Route path="/" element={<Bookshelf available={0} />} />
```

Comment:
Confusing prop naming 
when available = false, you're displaying available books

SUGGESTION: 
<Route path="/" element={<Bookshelf claimed={0} />} />